test_project/fastpg/core.py

```
@@ -270,9 +153,13 @@ async def _execute_query(self, func) -> None:
                 query=self.query, values=self.query_param_values)
             self.query_executed = True
         except Exception as e:
+            try:
+                sqlstate = e.sqlstate
+            except AttributeError:
+                raise e
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)